app/logic/searchUsers.py

```diff
@@ -23,4 +25,7 @@ def searchUsers(query):
                 for participant in results:
                     if participant not in resultsDict:
                         resultsDict[f"{participant.firstName} {participant.lastName} ({participant.username})"] = f"{participant.firstName} {participant.lastName} ({participant.username})"
+                        if phoneNumber=="true":
+                            resultsDict[f"{participant.username} phoneNumber"] = participant.phoneNumber
```

Comment:
I'm still a little confused why we have a dictionary with weird keys like "first last username" and "username phone", instead of just a dictionary with "username" as a key and a dictionary of all of the user's properties as the value.